### PR TITLE
fix(facial-expressions): load worker as a blob

### PR DESCRIPTION
**Pull request by gabiborlea**
Load the facial expression worker code as a blob and then use its URL for creating the worker.

### CODE REVIEW

File: react/features/facial-recognition/actions.js [hunk 1]
```diff
@@ -56,7 +56,21 @@ export function loadWorker() {
 
             return;
         }
-        worker = new Worker('libs/facial-expressions-worker.min.js', { name: 'Facial Expression Worker' });
+        let baseUrl = '';
+        const app: Object = document.querySelector('script[src*="app"]');
```
Comment (saghul) [inline react/features/facial-recognition/actions.js:60]: Can you put the full JS bundle URL? Including min.js that is.

File: react/features/facial-recognition/actions.js [hunk 2]
```diff
@@ -56,7 +56,21 @@ export function loadWorker() {
...
+
+        if (app) {
+
```
Comment (saghul) [inline react/features/facial-recognition/actions.js:63]: unnecessary newline

**Review by saghul — COMMENTED** [review]
Left a couple of comments.

**Review by saghul — APPROVED** [review]

**Comment by gabiborlea** [comment]
Jenkins please test this please.

**Pull request merged**